wallet/src/wallet.rs

```diff
@@ -45,6 +45,14 @@ pub trait CapeWalletBackend<'a>: WalletBackend<'a, CapeLedger> {
         asset: &AssetDefinition,
     ) -> Result<Option<Erc20Code>, CapeWalletError>;
 
+    /// Wait until the EQS reflects the associated ERC20 code of the CAPE asset, or the `timeout`
+    /// (in seconds) is reached.
+    async fn wait_for_wrapped_erc20_code(
+        &mut self,
+        asset: &AssetDefinition,
+        timeout: Option<u64>,
```

Comment:
We should probably be taking a `Duration` here and converting seconds to `Duration` at a higher level (like the API route handler)